src/3rd_party/apache-log4cxx-0.10.0/autom4te.cache/requests

```diff
@@ -15,63 +15,63 @@
                         'configure.in'
                       ],
                       {
+                        'AM_MAKEFILE_INCLUDE' => 1,
```

Comment:
@LuxoftAKutsan, Yes, but it has been changed during RC merge. So this PR will revert this changes.